include/core/truthTable/truth_table.hpp

```diff
@@ -150,6 +181,10 @@ namespace syrec {
             return cubeMap.begin()->second.size();
         }
 
+        auto findCubeInteger(const std::uint64_t number, const std::size_t bw) -> decltype(cubeMap.cbegin()) {
```

Comment:
```suggestion
        auto find(const std::uint64_t number, const std::size_t bw) -> decltype(cubeMap.cbegin()) {
```
Just a minor suggestion, since the intent already becomes clear from the parameters of the function.

Reply:
done 👍🏼